deltachat-ios/Controller/Settings/SettingsViewController.swift

```diff
@@ -572,7 +572,7 @@ internal final class SettingsViewController: UITableViewController, ProgressAler
     }
 
     private func showVideoChatInstance() {
-        let videoInstanceController = SettingsVideoChatViewController(dcContext: dcContext)
+        let videoInstanceController = VideoChatViewController(dcContext: dcContext)
```

Comment:
better call this `VideoChatInstanceViewController` or `VideoChatSettingsViewController` or so, otherwise, at a first glance one would think that this shows the video etc.